docs/book/driver-deployment/prerequisites.md

```diff
@@ -0,0 +1,156 @@
+| CNS-DATASTORE | ![CNS-DATASTORE-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/CNS-DATASTORE-USAGE.png)  |
+| CNS-VM | ![CNS-VM-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/CNS-VM-USAGE.png)  |
+| CNS-SEARCH-AND-SPBM | ![CNS-SEARCH-AND-SPBM-USAGE](https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/docs/images/CNS-SEARCH-AND-SPBM-USAGE.png)  |
+
+## Setting up the management network <a id="setup_management_network"></a>
+
+By default, CPI and CSI Pods are scheduled on k8s master nodes. In this case, for non-topology aware Kubernetes clusters, it is sufficient to provide the k8s master node(s) credentials to the vCenter that this cluster runs on.
+
+For topology aware clusters, every k8s node needs to discover its topology by communicating with the vCenter. This is needed to utilize the topology aware provisioning and late binding feature.
+
+Refer the CSI Deployment section to understand how to provide a kubernetes node with vCenter credentials.
+
+## Virtual Machine Configuration <a id="vm_configuration"></a>
+
+Make sure to configure all the VMs that form the kubernetes cluster with the following:
+
+- We recommend using the VMware Paravirtual SCSI controller for Primary Disk on the Node VMs.
+- Set the `disk.EnableUUID` parameter to `TRUE` for each node VM. This step is necessary so that the VMDK always presents a consistent UUID to the VM, thus allowing the disk to be mounted properly.
+  - This can be done on the VirtualCenter User Interface by: right clicking on the VM → Edit Settings → VM Options → Advanced → Edit Configuration.
+- VM Hardware version must be 15 or higher.
+  - This can be done on the VirtualCenter User Interface by: right clicking on the VM → Compatibility → Upgrade VM Compatibility.
+
+The VMs can also be configured by using the govc command line tool.
+
+- Install govc on your devbox/workstation.
+- Get the VM Paths
+
+    ```bash
+    $ export GOVC_INSECURE=1
+    $ export GOVC_URL='https://<VC_Admin_User>:<VC_Admin_Passwd>@<VC_IP>'
+
+    $ govc ls
+    /<datacenter-name>/vm
+    /<datacenter-name>/network
+    /<datacenter-name>/host
+    /<datacenter-name>/datastore
+
+    // To retrieve all Node VMs
+    $ govc ls /<datacenter-name>/vm
+    /<datacenter-name>/vm/<vm-name1>
+    /<datacenter-name>/vm/<vm-name2>
+    /<datacenter-name>/vm/<vm-name3>
+    /<datacenter-name>/vm/<vm-name4>
+    /<datacenter-name>/vm/<vm-name5>
+    ```
+
+- Enable disk UUID
+
+    Run the below command for all Node VMs part of the kubernetes cluster.
+
+    ```bash
+    govc vm.change -vm '/<datacenter-name>/vm/<vm-name1>' -e="disk.enableUUID=1"
+    ```
+  
+- Upgrade VM hardware version of node VMs to 15 or higher.
+
+  Run the below command for all Node VMs part of the kubernetes cluster.
```

Comment:
s/all Node VMs part of/all Node VMs that are part of

Reply:
done